### DISCUSSION TITLE
Browsing playlists or albums

### DISCUSSION THREAD

**Discussion by pookieman** [alandtse/alexa_media_player#2759] [Q&A]
Hi

Is it possible to browse/navigate music in the same way as I do with the spotify integrations with HASS?

So I'd like to be able to see the media library like I do with Spotify but with Amazon music..

<img width="692" alt="image" src="https://github.com/user-attachments/assets/883e417d-f0e8-41d5-bd2e-05c62a03ae6f" />

I can playback music but I have to select the music from my phone first.

thanks

**Comment by danielbrunt57** [comment]
Where are you seeing this?
![image](https://github.com/user-attachments/assets/cba4f7ec-8301-435b-8b7c-ab8b53354edd)

**Comment by danielbrunt57** [comment]
[My first reaction to your discussion]()
Amazon Music is unrelated to Alexa Media Player which simply creates `media_player` devices as derived from querying your Amazon Alexa network of Echo devices.  I believe Spotify/SpotifyPlus is an integration which interacts with Spotify Music and media player devices it can use are external to it. e.g. Sonos, AMP, etc. What you are looking for is an equivalent to SpotifyPlus for Amazon Music which I believe is well outside the scope of Alexa Media Player.

**Comment by danielbrunt57** [comment]
Reference: [Has anyone managed to integrate amazon music into home assistant](https://community.home-assistant.io/t/has-anyone-managed-to-integrate-amazon-music-into-home-assistant/216908)

Also: [Google: home assistant and amazon music library](https://www.google.com/search?q=home+assistant+and+amazon+music+library&oq=home+assistant+and+amazon+music+library&gs_lcrp=EgRlZGdlKgYIABBFGDkyBggAEEUYOdIBCDkyNDFqMGo5qAIAsAIB&sourceid=chrome&ie=UTF-8)
